internal/functions/deploy/testdata/modules/imports.ts

```diff
@@ -0,0 +1,90 @@
+import {
+  Component
+} from '@angular2/core';
+import defaultMember from "module-name";
+import   *    as name from "module-name  ";
+import   {  member }   from "  module-name";
+import { member as alias } from "module-name";
+import { member1 , member2 } from "module-name";
+import { member1 , member2 as alias2 , member3 as alias3 } from "module-name";
+import {
+  Component
+} from '@angular2/core';
+import defaultMember from "$module-name";
+import defaultMember, { member, member } from "module-name";
+import defaultMember, * as name from "module-name";
+
+import   *    as name from "module-name  "
+import   {  member }   from "  module-name"
+import { member as alias } from "module-name"
+import { member1 , member2 } from "module-name"
+import { member1 , member2 as alias2 , member3 as alias3 } from "module-name"
+import {
+  Component
+} from '@angular2/core'
+import defaultMember from "$module-name"
+import defaultMember, { member, member } from "module-name"
+import defaultMember, * as name from "module-name"
+
+import "module-name";
+import React from "react"
+import { Field } from "redux-form"
+import "module-name";
+
+import {
+    PlaneBufferGeometry,
+    OctahedronGeometry,
+    TorusBufferGeometry
+} from '../geometries/Geometries.js';
+
+import {
+    PlaneBufferGeometry,
+    OctahedronGeometry,
+    TorusBufferGeometry
+} from '../geometries/Geometries.js'
```

Comment:
are we also allowing importing files outside `supabase/functions` directory? eg: `../../../geometries/Geometries.js`? If so, would be good to add a test scenario for it too.

Reply:
Not possible at the moment because the workdir is always set to parent of supabase directory. So that's the top level it can reach. But once we fix the workdir issue, triple `..` should also work.

Reply:
Can users continue to add files outside of supabase directory to import map and have those be included?